src/corppa/poetry_detection/core.py

```diff
@@ -2,9 +2,17 @@
 Custom data type for poetry excerpts identified with the text of PPA pages.
 """
 
-from dataclasses import asdict, dataclass
+from dataclasses import asdict, dataclass, field, replace
 from typing import Any, Optional
 
+# Would upper casing be better?
+DETECTION_PFX_TABLE = {
```

Comment:
I would rename to just DETECTION_METHODS or something like that, since it's serving both purposes (list of valid methods as well as the id prefix).